src/display/api.js

```diff
@@ -784,7 +787,10 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
         this.stats.time('Page Request');
         this.transport.messageHandler.send('RenderPageRequest', {
           pageIndex: this.pageNumber - 1,
-          intent: renderingIntent
+          intent: renderingIntent,
+          renderInteractiveForms:
+            (params && params.renderInteractiveForms === true ? true :
+             /* Default */ false),
```

Comment:
I think that readability would be improved here if you instead defined an intermediate `renderInteractiveForms` variable above, similar to the `renderingIntent`. E.g.
```javascript
  var renderInteractiveForms = (params.renderInteractiveForms === true ?
                                true : /* Default */ false);

  ...

  this.transport.messageHandler.send('RenderPageRequest', {
    pageIndex: this.pageNumber - 1,
    intent: renderingIntent,
    renderInteractiveForms: renderInteractiveForms,
  });
```